config/helm/manager_extra_volume.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      volumes:
+        - name: CONTROLLER_MANAGER_EXTRA_VOLUME
```

Comment:
Where is `CONTROLLER_MANAGER_EXTRA_VOLUME` and `CONTROLLER_MANAGER_EXTRA_VOLUME` defined?

Reply:
`make helm` command runs `kustomize` in `/config/helm` directory and kustommize generates all manifests including this `CONTROLLER_MANAGER_EXTRA_VOLUME`. The output of the kustomize will be sent to [helm script](https://github.com/TykTechnologies/tyk-operator/blob/feature/TT-9291/tls-connection-custom-ca/hack/helm/pre_helm.go).

In the script, we replaces all hardcoded values with their helm templating correspondence. For `CONTROLLER_MANAGER_EXTRA_VOLUME`, it is replaced here: https://github.com/TykTechnologies/tyk-operator/blob/61fce465679c7528c74aae960139e9ab238ad308/hack/helm/pre_helm.go#L25

Reply:
Got it. Thanks!